packages/flutter/lib/src/material/slider.dart

```diff
@@ -44,6 +44,36 @@ enum _SliderType { material, adaptive }
 ///
 /// Used to select from a range of values.
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold}
+///
+/// This is what it looks like when run:
+///
+/// ![A slider widget, consisting of 5 divisions and showing the default value
+/// indicator.](https://flutter.github.io/assets-for-api-docs/assets/material/slider.png)
+///
+/// This sample shows creation of a [Slider] widget with 5 divisions.
+///
+/// ```dart
+/// double _sliderDiscreteValue = 20;
```

Comment:
As with the earlier example; might be slightly clearer as _currentSliderValue

Reply:
Done